client/jetpack-cloud/sections/agency-dashboard/sites-overview/types.ts

```diff
@@ -18,7 +19,8 @@ export type AllowedStatusTypes =
 	| 'warning'
 	| 'success'
 	| 'disabled'
-	| 'critical';
+	| 'critical'
+	| null;
```

Comment:
In which cases we might have `null` as a value here?

Reply:
This was a failed Friday attempt to be able to not have a status for the StatsNode.